docs/the-new-architecture/cxx-custom-types.md

```diff
@@ -0,0 +1,284 @@
+---
+id: cxx-custom-types
+title: Supporting Custom C++ Types
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/TabsConstants';
+import NewArchitectureWarning from '../\_markdown-new-architecture-warning.mdx';
+
+By default C++ Turbo Native Modules support [bridging functionality](https://github.com/facebook/react-native/tree/main/ReactCommon/react/bridging) for most `std::` standard types.
```

Comment:
As per all the others pages in the New Architecture guides, we should add the yellow experimental banner.

```
import NewArchitectureWarning from '../\_markdown-new-architecture-warning.mdx';

<NewArchitectureWarning/>
```

Reply:
Agree!